basicsettings.ml

```diff
@@ -24,14 +24,35 @@ let concurrent_server = Settings.add_bool ("concurrent_server", true, `System)
 let printing_types = Settings.add_bool ("printing_types", true, `User)
 
 (** Name of the file containing the prelude code. *)
-let prelude_file = 
+let prelude_file =
   let prelude_dir = match Utility.getenv "LINKS_LIB" with
-      None -> Filename.dirname Sys.executable_name
+    (* If user defined LINKS_LIB then it takes the highest priority *)
     | Some path -> path
+    | None ->
+        (* If LINKS_LIB is not defined then we search in current directory *)
+        let current_dir = Filename.dirname Sys.executable_name in
```

Comment:
I'll change the variable name to something more accurate.

> I just found this dbuenzli/logs#14 which suggests that that would not work if links is bytecompiled.

Up until two days ago I used to byte-compile links and this code worked (it was there before my changes). So that would be surprising...